### ISSUE TITLE
Add an @import support

### ISSUE THREAD

**Issue by zoxon** [MaciekBaron/sublime-list-stylesheet-vars#18]
The plugin works only in the current file, please do the treatment variables from other files connected using @import all variables were taken

**Comment by MaciekBaron** [comment]
`@import` support is already in place, can you clarify what exactly is not working?

**Comment by zoxon** [comment]
Hello.
I use the stylus.
I have a file **variables.styl**, the index file for mixins is **mixins.styl** and the main stylesheet.
In **mixins.styl** connect everything I need including **variables.styl**

```
@import 'nib'
@import '/lib/variables'
@import '/lib/mixins/font'
@import '/lib/mixins/normalize'
```

At the beginning of the main file styles connected **mixins.jade**, but the variables are defined in **variables.styl** missing.

Maybe it's due to the fact that the variables are in the third file. (_main.styl → mixins.styl → variables.styl_)

Thank you

**Comment by MaciekBaron** [comment]
Yes, at the moment import support only works for one level of depth, that is, it will read imports in the files you are working on but it will not continue reading files that are imported by your imported files. Getting a bit of a inception situation here...

![](http://cdn.meme.li/instances/600x/54961042.jpg)

This could be added in the future, perhaps if I introduce some caching to improve performance. At the moment the variables are read when you execute the command. Definitely a nice feature to work on.
